Consul/Consul.csproj

```diff
@@ -2,39 +2,19 @@
 
   <PropertyGroup>
     <VersionPrefix>1.6.10.1</VersionPrefix>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.0;net461</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT' Or '$(Channel)' == 'mono'">$(TargetFrameworks);net461</TargetFrameworks>
```

Comment:
`dotnet5` sdk can build `net461` on any OS, so the is no need for this conditional definition of `TargetFramerworks`.